### PR TITLE
Adjust Symfony ruleset

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: src/RuleSet.php [hunk 1]
```diff
@@ -157,9 +168,11 @@
             'modernize_types_casting' => true,
             'no_alias_functions' => true,
             'no_homoglyph_names' => true,
-            'non_printable_character' => true,
+            'non_printable_character' => [
+                // @TODO: on 3.0 switch whole rule to `=> true`, currently we use custom config that will be default on 3.0
+                'use_escape_sequences_in_strings' => true,
+            ],
             'php_unit_construct' => true,
-            'php_unit_dedicate_assert' => true,
```
Comment (keradus) [inline src/RuleSet.php]: As by default it applies changes for PHPUnit 5.0, but Sf can apply on Sf only 4.8.
As this is part of migration ruleset, it shall not be part of style ruleset of `@Symfony` (and Sf is already using PHPUnit migration ruleset pinned to PHPUnit 4.8).

File: src/RuleSet.php [hunk 2]
```diff
@@ -157,9 +168,11 @@
             'modernize_types_casting' => true,
             'no_alias_functions' => true,
             'no_homoglyph_names' => true,
-            'non_printable_character' => true,
+            'non_printable_character' => [
+                // @TODO: on 3.0 switch whole rule to `=> true`, currently we use custom config that will be default on 3.0
+                'use_escape_sequences_in_strings' => true,
```
Comment (keradus) [inline src/RuleSet.php]: Sf is following this new feature, which is disabled by default on 2.x and will be enabled by default on 3.x

File: src/RuleSet.php [hunk 3]
```diff
@@ -109,7 +109,18 @@
...
+                // @TODO: on 3.0 switch whole rule to `=> true`, currently we use custom config that will be default on 3.0
+                'tags' => [
+                    'method',
+                    'param',
+                    'property',
+                    'return',
+                    'throws',
+                    'type',
+                    'var',
+                ],
```
Comment (keradus) [inline src/RuleSet.php]: Sf is following this new feature, which is disabled by default on 2.x and will be enabled by default on 3.x

File: src/RuleSet.php (hunk 1 above)
Comment (SpacePossum) [reply src/RuleSet.php]: maybe add this reason as a comment to the code?

Comment (keradus) [reply src/RuleSet.php]: it is not changing to `'php_unit_dedicate_assert' => false`, it is dropping the line.
hard to add comment to line that does not exist anymore ;)

Comment (SpacePossum) [reply src/RuleSet.php]: haha fair enough :)
:ship:

**Comment by SpacePossum** [comment]
looks good, however sadly the tests are failing

**Pull request merged**